js/modules/k6/experimental/fs/file.go

```diff
@@ -8,25 +8,28 @@ import (
 
 // file is an abstraction for interacting with files.
 type file struct {
-	path string
+	path string `js:"path"`
```

Comment:
We made this change a long time ago, but if my memory is correct, this is to ensure that although the struct and those fields are private, they are correctly serialized in the object at the JS runtime level; so that when the public object `Stat()` method needs access to it, it does 🙇🏻